client/core/core.go

```diff
@@ -3904,16 +3903,18 @@ func (c *Core) prepareTrackedTrade(dc *dexConnection, form *TradeForm, crypter e
 	// Send a low-priority notification.
 	corder := tracker.coreOrder()
 	if !form.IsLimit && !form.Sell {
+		ui := wallets.quoteWallet.Info().UnitInfo
 		subject, details := c.formatDetails(TopicYoloPlaced,
-			float64(corder.Qty)/conversionFactor, unbip(form.Quote), tracker.token())
+			ui.ConventionalString(corder.Qty), ui.Conventional.Unit, tracker.token())
 		c.notify(newOrderNote(TopicYoloPlaced, subject, details, db.Poke, corder))
 	} else {
 		rateString := "market"
 		if form.IsLimit {
-			rateString = strconv.FormatFloat(float64(corder.Rate)/conversionFactor, 'f', 8, 64)
+			rateString = strconv.FormatFloat(float64(corder.Rate)/calc.RateConversionFactor, 'f', 8, 64)
```

Comment:
For the purposes of showing the rate to users, will this need to also be divided by `ui.Conventional.Unit`? If I look at the `dcr_btc` market for our default siment I will see `Rate Step: 0.000001 BTC`. If ethereum was the quote asset, it would need one more zero to be correct, as one unit will be 1e9 of one eth.

Reply:
Rates are technically in units of quote/base, e.g. `BTC/DCR` but in units of sats and atoms. Some places we simply list it as the quote asset's symbol like you said, which I feel like is common but technically wrong.

But you did just make me realize that there is a difference between 1) converting the rate from the fractional to integer form for float-free comms, and 2) displaying a conventional rate or parsing one from input. For example, if someone was trading 1 ETH for 1 DCR on the DCR-ETH market, the exchange rate they type into the form would just be `1`. But for official purposes, in atomic units, it would be `1e9/1e8 = 10`, (which would be encoded as `10 * 1e8 = 1e9`). So I guess when displaying and parsing rates, we need to be scaling by the ratio of the conventional conversion factors. Crap.

Reply:
Fixed in e6931e9

There are actually three rates encodings of interest. One is the atomic ratio, which is `quoteQty / baseQty` Quantities for this rate are in atomic units. Then there is the message-rate encoding, which is roughly `uint64(atomicRatio * 1e8)`. Then there is the conventional ratio, which is similar to the atomic ratio, but in conventional units. The relationship between the atomic ratio and the conventional ratio is

`conventionalRatio = atomicRatio * (baseConversionFactor / quoteConversionFactor)`

where the conversion factors are the factors for the conventional units.

Users will want to work with the `conventionalRatio`, but we'll always prefer to encode as message-rate on the backend. `SimpleOrder` is updated to use the message-rate encoding too.